src/tribler-core/tribler_core/modules/metadata_store/community/gigachannel_community.py

```diff
@@ -29,6 +30,9 @@
 MAGIC_GIGACHAN_VERSION_MARK = b'\x01'
 
 
+loop = asyncio.get_event_loop()
```

Comment:
Why is this here? As far as I can tell, you're not using it anywhere.

Reply:
:face_with_head_bandage: